stripe/src/main/java/com/stripe/android/StripeApiHandler.java

```diff
@@ -133,6 +135,156 @@ static Source createSource(
                 null);
     }
 
+
+    @Nullable
+    static PaymentIntent confirmPaymentIntent(
```

Comment:
nit: putting these methods between `createSource(...)` and `createSource(...)` seems like non-ideal organization. I'd either move them up 1 or down 2.

Reply:
Are you going to add javadocs to match the create/retrieve Source methods?